ui/pages/swaps/build-quote/build-quote.test.js

```diff
@@ -130,7 +132,17 @@ describe('BuildQuote', () => {
   it('renders the block explorer link, only 1 verified source', () => {
     const mockStore = createSwapsMockStore();
     mockStore.swaps.toToken.occurances = 1;
-    const store = configureMockStore(middleware)(mockStore);
+    const store = configureMockStore(middleware)({
+      ...mockStore,
+      metamask: {
+        ...mockStore.metamask,
+        ...mockNetworkState({
+          chainId: CHAIN_IDS.MAINNET,
+          blockExplorerUrl:
+            'https://etherscan.io/token/0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48',
```

Comment:
Shouldn't this just be `https://etherscan.io`

Reply:
yes this looks off, let me check

Reply:
fixed in https://github.com/MetaMask/metamask-extension/pull/26433/commits/97104e242472bd9263e59dbb9714271169ef0972